DSCPullServerAdmin/src/CmdLets/RemoveDSCPullServerAdminRegistration.cs

```diff
@@ -6,16 +6,19 @@
 
 namespace DSCPullServerAdmin.src.CmdLets
 {
-    [Cmdlet(VerbsCommon.Remove,
-        "DSCPullServerAdminRegistration")]
+    [Cmdlet(VerbsCommon.Remove, "DSCPullServerAdminRegistration", SupportsShouldProcess = true, ConfirmImpact = ConfirmImpact.High, DefaultParameterSetName = "Id")]
```

Comment:
Could you create a new line after every comma and style like:
```
[Cmdlet(VerbsCommon.Remove,
        "DSCPullServerAdminRegistration",
        SupportsShouldProcess = true,
        ConfirmImpact = ConfirmImpact.High,
        DefaultParameterSetName = "Id")]
```
Please do for all occurrences.